librz/include/rz_core.h

```diff
@@ -801,6 +801,45 @@ RZ_API char *rz_core_bin_method_build_flag_name(RzBinClass *cls, RzBinSymbol *me
 RZ_API char *rz_core_bin_method_flags_str(ut64 flags, int mode);
 RZ_API bool rz_core_pdb_info(RzCore *core, const char *file, PJ *pj, int mode);
 
+typedef enum {
+	RZ_CORE_STRING_KIND_UNKNOWN,
+	RZ_CORE_STRING_KIND_ASCII,
+	RZ_CORE_STRING_KIND_WIDE16, // No UTF/Unicode encoding
+	RZ_CORE_STRING_KIND_WIDE32, // NO UTF/Unicode encoding
+	RZ_CORE_STRING_KIND_UTF8,
+	RZ_CORE_STRING_KIND_UTF16,
+	RZ_CORE_STRING_KIND_UTF32,
+} RzCoreStringType;
```

Comment:
Should be either `RzCoreStringType` + `*_TYPE_*` or `RzCoreStringKind` + `_KIND_`.

Reply:
@thestr4ng3r so is it fine for you that it *partly* overlaps with `RzBinString` structure?

Reply:
`RzBinString` partially overlapping with `RzCoreString` does make sense imo since `RzBinString` can mean strings that are directly defined by a binary (like in a jvm string pool or something) vs. general strings coming from heuristic search or other things in core.